tensorbay/opendataset/_utility/voc.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Common methods for loading VOC formatted datasets."""
+
+import os
+from typing import Set
+
+from tensorbay.dataset import Data
+from tensorbay.label import LabeledBox2D
+
+try:
+    import xmltodict
+except ModuleNotFoundError:
+    from tensorbay.opendataset._utility.mocker import xmltodict  # pylint:disable=ungrouped-imports
+
+
+def get_voc_detection_data(
+    stem: str, image_path: str, annotation_path: str, boolean_attributes: Set[str]
+) -> Data:
+    """Get all information of the datum corresponding to voc-like label files.
+
+    Arguments:
+        stem: The filename without extension of the data.
+        image_path: The path of the image directory.
+        annotation_path: The path of the annotation directory.
+        boolean_attributes: The boolean attributes of the data.
+
+    Returns:
+        Data: class:`~tensorbay.dataset.data.Data` instance.
+
+    """
+    data = Data(os.path.join(image_path, f"{stem}.jpg"))
+    box2d = []
+    with open(os.path.join(annotation_path, f"{stem}.xml"), "r", encoding="utf-8") as fp:
+        objects = xmltodict.parse(fp.read())["annotation"]["object"]
```

Comment:
Add typehint `Any` for the result of `xmltodict.parse(fp.read())` to avoid Pyright error. Then get the `["annotation"]["object"]`.